CollapseLauncher/XAMLs/MainApp/Pages/StarRailGameSettingsPage.xaml.cs

```diff
@@ -78,8 +80,16 @@ private void LoadPage()
             GameSettingsApplyGrid.Translation = new System.Numerics.Vector3(0, 0, 64);
 
             InheritApplyTextColor = ApplyText.Foreground;
+            
+            // A/B Testing as of 2023-12-26 (HSR v1.6.0)
+            object? abValue = Registry.GetValue(@"HKEY_CURRENT_USER\Software\Cognosphere\Star Rail", _AbValueName, null);
```

Comment:
Put this line under `#nullable enable` context and close the nullable at line 90

Reply:
Fixed in 9c5c173